certified-connectors/IronDefense/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2246 @@
+        "tags": [
+          "IronApi"
+        ]
+      }
+    },
+    "/IronApi/GetAlertNotifications": {
+      "post": {
+        "summary": "Get alert notifications",
+        "description": "Allows a client to retrieve alert notifications from IronDefense without pulling duplicate messages that have already been ingested. The default request rate limit on this endpoint is 10 requests per second.",
+        "operationId": "GetAlertNotifications",
+        "responses": {
+          "200": {
+            "description": "A successful response.",
+            "schema": {
+              "$ref": "#/definitions/alertGetAlertNotificationsResponse"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/alertGetAlertNotificationsRequest"
+            }
+          }
+        ],
+        "tags": [
+          "IronApi"
+        ]
+      }
+    },
+    "/IronApi/GetAlerts": {
+      "post": {
+        "summary": "Get alerts",
+        "description": "Allows a client to retrieve IronDefense alerts in an environment. The response can be filtered based on the alert field parameters and limited to a given number of alerts. The default request rate limit on this endpoint is 10 requests per second.",
+        "operationId": "GetAlerts",
+        "responses": {
+          "200": {
+            "description": "A successful response.",
+            "schema": {
+              "$ref": "#/definitions/alertGetAlertsResponse"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/alertGetAlertsRequest"
+            }
+          }
+        ],
+        "tags": [
+          "IronApi"
+        ]
+      }
+    },
+    "/IronApi/GetDomeNotifications": {
+      "post": {
+        "summary": "Get IronDome notifications",
+        "description": "Allows a client to retrieve dome notifications from IronDefense without pulling duplicate messages that have already been ingested. The default request rate limit on this endpoint is 10 requests per second.",
+        "operationId": "GetDomeNotifications",
+        "responses": {
+          "200": {
+            "description": "A successful response.",
+            "schema": {
+              "$ref": "#/definitions/domeGetDomeNotificationsResponse"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/domeGetDomeNotificationsRequest"
+            }
+          }
+        ],
+        "tags": [
+          "IronApi"
+        ]
+      }
+    },
+    "/IronApi/GetEvent": {
+      "post": {
+        "summary": "Get event",
```

Comment:
@marcus-defazio It can be `"Get an event"`!!

Reply:
Changed.